test/integration/testMaxLobSize.js

```diff
@@ -95,6 +98,52 @@ if (process.env.RUN_MANUAL_TESTS_ONLY === 'true'){
       });
     });
 
+    describe('test put get max LOB size', function () {
+      before(async function () {
+        connection = testUtil.createConnection();
+        await testUtil.connectAsync(connection);
+        await testUtil.executeCmdAsync(connection, createTable);
+        await testUtil.executeCmdAsync(connection, allowLobExternalScan);
+      });
+
+      after(async () => {
+        await testUtil.executeCmdAsync(connection, removeStage);
+        await testUtil.dropTablesIgnoringErrorsAsync(connection, [tableName]);
+        await testUtil.destroyConnectionAsync(connection);
+      });
+
+      it('test - copy to stage where size > 16MB', function (done){
+        const rowData = mediumSizeTableData.C1 + ',' + mediumSizeTableData.C2 + ',' + mediumSizeTableData.C3 + '\n';
```

Comment:
why do we use medium, not max?

Reply:
Because medium is already > 16MB and it is significantly faster.